ansible/roles/bootstrap-industrial-edge/README.md

```diff
@@ -1,31 +1,73 @@
+| Vault CLI | The Vault CLI is a single static binary. It is a thin wrapper around the HTTP API. Every CLI command maps directly to the HTTP API internally. | RHEL: <br> **sudo yum-config-manager --add-repo https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo**<br> **sudo yum -y install vault** <br><br> Fedora: <br> **sudo dnf config-manager --add-repo https://rpm.releases.hashicorp.com/fedora/hashicorp.repo** <br> **sudo dnf -y install vault** |
 
-A description of the settable variables for this role should go here, including any variables that are in defaults/main.yml, vars/main.yml, and any variables that can/should be set via parameters to the role. Any variables that are read from other roles and/or the global scope (ie. hostvars, group vars, etc.) should be mentioned here as well.
 
-Dependencies
+## Role Variables
 ------------
 
-A list of other roles hosted on Galaxy should go here, plus any details in regards to parameters that may need to be set for other roles, or variables that are used from other roles.
+Most of the variables will be dynamically set for you in this role. Variables that we will be looking for are:
 
-Example Playbook
-----------------
+| Variable | Description | Default Value |
+| --------- | ---------- | ---------- | 
+pattern_repo_dir:  | Pattern directory.  We assume that you start the execution of the ansile role in the pattern working cloned directory. |  "{{ lookup('env', 'PWD') }}" |            
+| argo_target_namespace: | Target namespace for ArgoCD | manuela-ci                                           |
+| pattern: | Name of the validated pattern | industrial-edge |                                                         
+| component: | Name of the component to deploy |datacenter |                                                       
+| secret_name: | Name of the ArgoCD secret in OpenShift | "argocd-env" |                                                     
+| values_secret: | Location of the values-secret.yaml file | "{{ lookup('env', 'HOME') }}/values-secret.yaml" |                                                                    | values_global: | Location of the values-global.yaml file | "{{ pattern_repo_dir }}/values-secret.yaml" |             
+| kubeconfig: | Environment variable for KUBECONFIG | "{{ lookup('env', 'KUBECONFIG') }}"|
+| vault_init_file: | Init Vault file which will contain Vault tokens etc | "{{ pattern_repo_dir }}/common/pattern-vault.init"|                      
+| vault_ns: | Namespace for Vault | "vault"|                                                        
+| vault_pod: | Name of the initial Vault pod | "vault-0"|                                                       
+| vault_path: | Path to the Vault secrets for the Hub | "secret/hub"|                                                           
+| debug: | Whether or not to display debug info | False |
```

Comment:
If you do another pass at this maybe remove the whitespaces at the end

Reply:
Completed.